draft-ietf-add-ddr.md

```diff
@@ -322,6 +322,8 @@ Since client can receive DNS SVCB answers over unencrypted DNS, on-path
 attackers can prevent successful discovery by dropping SVCB packets. Clients
 should be aware that it might not be possible to distinguish between resolvers
 that do not have any Designated Resolver and such an active attack.
+To limit the impact of discovery queries being dropped either maliciously or
+unintentionally, clients can re-send their SVCB queries periodically.
```

Comment:
I'm not sure whether "can" or "MAY" is more appropriate here. I would go with the latter if we also are trying to signal that forwarders and recursives can and should expect some clients to repeat the query until a valid SVCB is returned.

Reply:
Going to a normative MAY is a bit overloaded with implying that it's not a SHOULD or MOST, so I wanted to avoid that in this security considerations section. Keeping it as "can" is noting a fact without suggesting strongly one way or the other.